staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation_test.go

```diff
@@ -693,8 +693,8 @@ func TestCelCompilation(t *testing.T) {
 				},
 			},
 			expectedResults: []validationMatcher{
-				invalidError("undefined field 'namespace'"),
-				invalidError("undefined field 'if'"),
+				noError(),
+				noError(),
```

Comment:
Since test name is "invalid for escaping", add a comment here explaining why this changed?

Reply:
Updated :)